jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/PodTemplateHandlerTest.java

```diff
@@ -140,13 +135,13 @@ public void podWithVolumeTemplateHandlerTest(){
         PodTemplateSpec podTemplateSpec = podTemplateHandler.getPodTemplate(config, null, images);
 
         //Assertion
-        assertEquals("test-account",podTemplateSpec.getSpec().getServiceAccountName());
-        assertFalse(podTemplateSpec.getSpec().getVolumes().isEmpty());
-        assertEquals("test",podTemplateSpec.getSpec()
-                .getVolumes().get(0).getName());
-        assertEquals("/test/path",podTemplateSpec.getSpec()
-                .getVolumes().get(0).getHostPath().getPath());
-        assertNotNull(podTemplateSpec.getSpec().getContainers());
+        assertThat(podTemplateSpec.getSpec().getServiceAccountName()).isEqualTo("test-account");
+        assertThat(podTemplateSpec.getSpec().getVolumes()).isNotEmpty();
+        assertThat(podTemplateSpec.getSpec()
+                .getVolumes().get(0).getName()).isEqualTo("test");
+        assertThat(podTemplateSpec.getSpec()
+                .getVolumes().get(0).getHostPath().getPath()).isEqualTo("/test/path");
+        assertThat(podTemplateSpec.getSpec().getContainers()).isNotNull();
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.